website/docs/openmp.md

```diff
@@ -0,0 +1,34 @@
+Enable or disable [OpenMP](https://en.wikipedia.org/wiki/OpenMP).
+
+```lua
+openmp "value"
+```
+If no value is set for a configuration, the toolsets default OpenMP option (usually Off) will be performed.
+
+### Parameters ###
+
+`value` is one of:
+
+| Value   | Description                                       |
+|---------|---------------------------------------------------|
+| On      | Turn on OpenMP.                                   |
+| Off     | Turn off OpenMP.                                  |
+
+### Applies To ###
+
+Project configurations.
+
+### Availability ###
+
+Require a version 5.0 alpha 16 or later. Currently only implemented for Visual Studio 2010+. As a workaround for other toolsets, you can use [buildoptions](buildoptions.md) like this:
```

Comment:
Technically, a build of alpha 17 or later.  Alpha 16 does not have this included.

Reply:
Do you mean `Require a version 5.0 beta 1 or later.`?

Reply:
Yup!